src/language/class-modifiers.md

```diff
@@ -151,20 +151,31 @@ class MockVehicle          // Can be implemented
 }
 ```
 
-To define a pure interface that others are free to implement, that cannot be
-inherited but is allowed to have abstract members, define an
-`abstract interface class`.
+### `abstract interface`
+
+The most common use for the `interface` modifier is to [combine](#combining-modifiers)
+it with [`abstract`](#abstract) for an `abstract interface class`.
+This defines a pure interface. 
```

Comment:
**suggestion:** Write in active voice per [GSG](https://developers.google.com/style/voice).
```suggestion
To create a pure interface, [combine](#combining-modifiers) the `interface` 
and [`abstract`](#abstract)  modifiers. 
The resulting `abstract interface class` defines a pure interface. 
```